src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -309,20 +368,28 @@
           :toolbar {:get-items (constantly toolbar-items)}}]]))})
 
 (react/defc NotebooksContainer
-  {:refresh
+  {;{:get-initial-state
+  ; (fn []
+  ;   {:cluster-status-gensym (gensym)})
+   :refresh
    (fn [{:keys [this]}]
-     (this :-get-clusters-list-if-whitelisted)
-     (this :-schedule-cookie-refresh-if-whitelisted))
-
+     (this :-get-clusters-list)
+     (this :-schedule-cookie-refresh))
    :render
    (fn [{:keys [props state this]}]
-     (let [{:keys [server-response show-create-dialog?]} @state
+     (let [{:keys [server-response show-create-dialog? cluster-status-gensym]} @state
            {:keys [clusters server-error]} server-response]
        [:div {:display "inline-flex"}
         (when show-create-dialog?
-          [ClusterCreator (assoc props :dismiss #(swap! state dissoc :show-create-dialog?)
-                                       :reload-after-create #(this :-get-clusters-list-if-whitelisted))])
-        [:div {} [:span {:data-test-id "spark-clusters-title" :style {:fontSize "125%" :fontWeight 500 :paddingBottom 10}} "Spark Clusters"]]
+          [ClusterCreator (assoc props :dismiss (fn []
+                                                  (swap! state #(-> % (assoc :cluster-status-gensym (gensym))
+                                                                         (dissoc :show-create-dialog?)))
+                                                  )
+                                       :reload-after-create #(this :-get-clusters-list))])
+        [:div {} [:span {:data-test-id "spark-clusters-title" :style {:fontSize "125%" :fontWeight 500 :paddingBottom 10 :marginLeft 10 }} "Spark Clusters"]]
+        [:div {:style {:margin 10 :fontSize "88%"} }
+         "Launch an interactive analysis environment based on Jupyter notebooks, Spark, and Hail.
+          This beta feature is under active development. See documentation here" [:a {:href (config/user-notebooks-guide-url) :target "_blank"} icons/external-link-icon]]
```

Comment:
Minor: should `here` be a link?

Reply:
I'll ask Brad!

Reply:
Definitely put the `here` inside the link. You should use `links/create-external` as well, in order to get the icon and target=_blank for free.